{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "045c6462",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting mysql-connector-python\n",
      "  Using cached mysql_connector_python-8.0.28-py2.py3-none-any.whl (342 kB)\n",
      "Collecting protobuf>=3.0.0\n",
      "  Using cached protobuf-3.19.4-py2.py3-none-any.whl (162 kB)\n",
      "Installing collected packages: protobuf, mysql-connector-python\n",
      "Successfully installed mysql-connector-python-8.0.28 protobuf-3.19.4\n"
     ]
    }
   ],
   "source": [
    "!pip install mysql-connector-python"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "64c8b035",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "###1번째 처리: 성공\n",
      "###2번째 처리: 성공\n",
      "###3번째 처리: 성공\n",
      "###4번째 처리: 성공\n",
      "###5번째 처리: 성공\n",
      "###6번째 처리: 성공\n",
      "###7번째 처리: 성공\n",
      "###8번째 처리: 성공\n",
      "###9번째 처리: 성공\n",
      "###10번째 처리: 성공\n",
      "###11번째 처리: 성공\n",
      "###12번째 처리: 성공\n",
      "###13번째 처리: 성공\n",
      "###14번째 처리: 성공\n",
      "###15번째 처리: 성공\n",
      "###16번째 처리: 성공\n",
      "###17번째 처리: 성공\n",
      "###18번째 처리: 성공\n",
      "###19번째 처리: 성공\n",
      "###20번째 처리: 성공\n",
      "###21번째 처리: 성공\n",
      "###22번째 처리: 성공\n",
      "###23번째 처리: 성공\n",
      "###24번째 처리: 성공\n",
      "###25번째 처리: 성공\n",
      "###26번째 처리: 성공\n",
      "###27번째 처리: 성공\n",
      "###28번째 처리: 성공\n",
      "###29번째 처리: 성공\n",
      "###30번째 처리: 성공\n",
      "###31번째 처리: 성공\n",
      "###32번째 처리: 성공\n",
      "###33번째 처리: 성공\n",
      "###34번째 처리: 성공\n",
      "###35번째 처리: 성공\n",
      "###36번째 처리: 성공\n",
      "###37번째 처리: 성공\n",
      "###38번째 처리: 성공\n",
      "###39번째 처리: 성공\n",
      "###40번째 처리: 성공\n",
      "###41번째 처리: 성공\n",
      "###42번째 처리: 성공\n",
      "###43번째 처리: 성공\n",
      "###44번째 처리: 성공\n",
      "###45번째 처리: 성공\n",
      "###46번째 처리: 성공\n",
      "###47번째 처리: 성공\n",
      "###48번째 처리: 성공\n",
      "###49번째 처리: 성공\n",
      "###50번째 처리: 성공\n",
      "###51번째 처리: 성공\n",
      "###52번째 처리: 성공\n",
      "###53번째 처리: 성공\n",
      "###54번째 처리: 성공\n",
      "###55번째 처리: 성공\n",
      "###56번째 처리: 성공\n",
      "###57번째 처리: 성공\n",
      "###58번째 처리: 성공\n",
      "###59번째 처리: 성공\n",
      "###60번째 처리: 성공\n",
      "###61번째 처리: 성공\n",
      "###62번째 처리: 성공\n",
      "###63번째 처리: 성공\n",
      "###64번째 처리: 성공\n",
      "###65번째 처리: 성공\n",
      "###66번째 처리: 성공\n",
      "###67번째 처리: 성공\n",
      "###68번째 처리: 성공\n",
      "###69번째 처리: 성공\n",
      "###70번째 처리: 성공\n",
      "###71번째 처리: 성공\n",
      "###72번째 처리: 성공\n",
      "###73번째 처리: 성공\n",
      "###74번째 처리: 성공\n",
      "###75번째 처리: 성공\n",
      "('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095648b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a41c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087047f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d575b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f56a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d570a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108704d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a44c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a41c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e5b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ed60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956eaf0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e2b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105ea5670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870f490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ef40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d574f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109561610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095645e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a49d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109561340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109561e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10924ebb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095615b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870f040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ed60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109561dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fa30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4d30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109455130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095645e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109561460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1084a4fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870f760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a13a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fb50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105ea5670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a46d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108704fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d577f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f53d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095647c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fa30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095646d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a42e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fcd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109560610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870f5b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095607c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109560e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4310>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1093c8d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109560b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957c670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f54f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108b1a760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a47f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4310>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957c820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957ce80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a46a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f57c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957cb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10899b6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a46a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10899b430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10899bee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a43a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a45e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095607c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10899b640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d577f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10899be50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109560b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a9730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f57f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a9460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a9f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a95b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fb20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a41f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fb50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a41f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108704520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d571c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109560be0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f51f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d573d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087046a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109560a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109560190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f51c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095605e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957cd30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d579d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957c940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957c2e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106f3b0a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957c760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d43340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a46d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956eee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10957ce20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f57f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1095609d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a76d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ec10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10924ebb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e8e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e1f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e1c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ea00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a73a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fcd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7310>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956edc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e4c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ee80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a73a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1093faeb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109564130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a49d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a70a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956eca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e3d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a06d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108b1a760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ebe0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e8b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108738550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10898c730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1087384f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a73a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ee80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10898c460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10898cf40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f55b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ee50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870fb20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a1a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108993790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10870f280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956ed00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f57c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10898cc40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089934f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e2b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a77f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108704d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089936a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4be0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a44c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a18e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4be0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0d30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x108d57ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956edc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a05b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a00d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e0a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a47f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10956e460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a7640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a0610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f5640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1096f52b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1089a4c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "###472번째 처리: 성공\n",
      "###473번째 처리: 성공\n",
      "###474번째 처리: 성공\n",
      "###475번째 처리: 성공\n",
      "###476번째 처리: 성공\n",
      "###477번째 처리: 성공\n",
      "###478번째 처리: 성공\n",
      "###479번째 처리: 성공\n",
      "###480번째 처리: 성공\n",
      "###481번째 처리: 성공\n",
      "###482번째 처리: 성공\n",
      "###483번째 처리: 성공\n",
      "###484번째 처리: 성공\n",
      "###485번째 처리: 성공\n",
      "###486번째 처리: 성공\n",
      "###487번째 처리: 성공\n",
      "###488번째 처리: 성공\n",
      "###489번째 처리: 성공\n",
      "###490번째 처리: 성공\n",
      "###491번째 처리: 성공\n",
      "###492번째 처리: 성공\n",
      "###493번째 처리: 성공\n",
      "###494번째 처리: 성공\n",
      "###495번째 처리: 성공\n",
      "###496번째 처리: 성공\n",
      "###497번째 처리: 성공\n",
      "###498번째 처리: 성공\n",
      "###499번째 처리: 성공\n",
      "###500번째 처리: 성공\n",
      "###501번째 처리: 성공\n",
      "###502번째 처리: 성공\n",
      "###503번째 처리: 성공\n",
      "###504번째 처리: 성공\n",
      "###505번째 처리: 성공\n",
      "###506번째 처리: 성공\n",
      "###507번째 처리: 성공\n",
      "###508번째 처리: 성공\n",
      "###509번째 처리: 성공\n",
      "###510번째 처리: 성공\n",
      "###511번째 처리: 성공\n",
      "###512번째 처리: 성공\n",
      "###513번째 처리: 성공\n",
      "###514번째 처리: 성공\n",
      "###515번째 처리: 성공\n",
      "###516번째 처리: 성공\n",
      "###517번째 처리: 성공\n",
      "###518번째 처리: 성공\n",
      "###519번째 처리: 성공\n",
      "###520번째 처리: 성공\n",
      "###521번째 처리: 성공\n",
      "###522번째 처리: 성공\n",
      "###523번째 처리: 성공\n",
      "###524번째 처리: 성공\n",
      "###525번째 처리: 성공\n",
      "###526번째 처리: 성공\n",
      "###527번째 처리: 성공\n",
      "###528번째 처리: 성공\n",
      "###529번째 처리: 성공\n",
      "###530번째 처리: 성공\n",
      "###531번째 처리: 성공\n",
      "###532번째 처리: 성공\n",
      "###533번째 처리: 성공\n",
      "###534번째 처리: 성공\n",
      "###535번째 처리: 성공\n",
      "###536번째 처리: 성공\n",
      "###537번째 처리: 성공\n",
      "###538번째 처리: 성공\n",
      "###539번째 처리: 성공\n",
      "###540번째 처리: 성공\n",
      "###541번째 처리: 성공\n",
      "###542번째 처리: 성공\n",
      "###543번째 처리: 성공\n",
      "###544번째 처리: 성공\n",
      "###545번째 처리: 성공\n",
      "###546번째 처리: 성공\n",
      "###547번째 처리: 성공\n",
      "###548번째 처리: 성공\n",
      "###549번째 처리: 성공\n",
      "###550번째 처리: 성공\n",
      "###551번째 처리: 성공\n",
      "###552번째 처리: 성공\n",
      "###553번째 처리: 성공\n",
      "###554번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Flood control exceeded. Retry in 11.0 seconds\n",
      "###555번째 처리: 오류500메시지/이벤트처리:send_message_bot(): Timed out\n",
      "###556번째 처리: 성공\n",
      "###557번째 처리: 성공\n",
      "###558번째 처리: 성공\n",
      "###559번째 처리: 성공\n",
      "###560번째 처리: 성공\n",
      "###561번째 처리: 성공\n",
      "###562번째 처리: 성공\n",
      "###563번째 처리: 성공\n",
      "###564번째 처리: 성공\n",
      "###565번째 처리: 성공\n",
      "###566번째 처리: 성공\n",
      "###567번째 처리: 성공\n",
      "###568번째 처리: 성공\n",
      "###569번째 처리: 성공\n",
      "###570번째 처리: 성공\n",
      "###571번째 처리: 성공\n",
      "###572번째 처리: 성공\n",
      "###573번째 처리: 성공\n",
      "###574번째 처리: 성공\n",
      "###575번째 처리: 성공\n",
      "###576번째 처리: 성공\n",
      "###577번째 처리: 성공\n",
      "###578번째 처리: 성공\n",
      "###579번째 처리: 성공\n",
      "###580번째 처리: 성공\n",
      "###581번째 처리: 성공\n",
      "###582번째 처리: 성공\n",
      "###583번째 처리: 성공\n",
      "###584번째 처리: 성공\n",
      "###585번째 처리: 성공\n",
      "###586번째 처리: 성공\n",
      "###587번째 처리: 성공\n",
      "###588번째 처리: 성공\n",
      "###589번째 처리: 성공\n",
      "###590번째 처리: 성공\n",
      "###591번째 처리: 성공\n",
      "###592번째 처리: 성공\n",
      "###593번째 처리: 성공\n",
      "###594번째 처리: 성공\n",
      "###595번째 처리: 성공\n"
     ]
    }
   ],
   "source": [
    "import mysql.connector\n",
    "import requests, json\n",
    "import time\n",
    "\n",
    "# 데이터베이스 컨넥션을 생성한다.\n",
    "mydb = mysql.connector.connect(\n",
    "    host=\"127.0.0.1\",\n",
    "    user=\"smartnqi\",\n",
    "    passwd=\"nwai1234!\",\n",
    "    database=\"smart\"\n",
    ")\n",
    "\n",
    "# 커서를 생성한다.\n",
    "cur = mydb.cursor()\n",
    "\n",
    "# 데이터를 조회할 쿼리문을 작성한다. \n",
    "sql = '''\n",
    "    select \n",
    "        'call' as dataType,\n",
    "        phone_no,\n",
    "        meastime,\n",
    "        networkId,\n",
    "        groupId,\n",
    "        currentTime,\n",
    "        timeline,\n",
    "        cellId,\n",
    "        currentCount,\n",
    "        ispId,\n",
    "        testNetworkType,\n",
    "        userInfo1,\n",
    "        userInfo2,\n",
    "        siDo,\n",
    "        guGun,\n",
    "        addressDetail,\n",
    "        udpJitter,\n",
    "        downloadBandwidth,\n",
    "        uploadBandwidth,\n",
    "        sinr,\n",
    "        isWifi,\n",
    "        latitude,\n",
    "        longitude,\n",
    "        bandType,\n",
    "        p_dl_earfcn,\n",
    "        p_pci,\n",
    "        p_rsrp,\n",
    "        NR_EARFCN,\n",
    "        NR_PCI,\n",
    "        NR_RSRP,\n",
    "        NR_SINR\n",
    "        from tb_ndm_data_measure  \n",
    "        where \n",
    "            meastime between 20211101000000000 and 20211101235959999\n",
    "            and userInfo1 = '경상남도-사천시-남양동'\n",
    "            \n",
    "        order by savetime\n",
    "    '''\n",
    "\n",
    "# 쿼리문을 실행한다.\n",
    "cur.execute(sql)\n",
    "row_headers=[x[0] for x in cur.description]\n",
    "res = cur.fetchall()\n",
    "\n",
    "# 데이터를 가져와서 JSON 데이터를 생성한다.\n",
    "for idx, row in enumerate(res):\n",
    "    tmpData = dict(zip(row_headers,row))\n",
    "    jsonData = json.dumps(tmpData)\n",
    "    try:\n",
    "        r = requests.post(\"http://localhost:8000/monitor/json/\", data=jsonData)\n",
    "        # 전송실패 시 오류값을 출력한다.\n",
    "        if r.status_code == 200:\n",
    "            print(f\"###{idx+1}번째 처리: 성공\")\n",
    "        else:\n",
    "            print(f\"###{idx+1}번째 처리: 오류{r.status_code}{r.text}\")\n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(str(e))\n",
    "\n",
    "#     # 초당 3개씩 전달하도록 한다. \n",
    "#     # 너무 다량의 데이터를 연속해서 보내면 서버단에서 부하로 오류가 발생한다. \n",
    "#     if idx % 3 == 0:\n",
    "#         time.sleep(1)\n",
    "                  "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7431c150",
   "metadata": {
    "collapsed": true
   },
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
